apps/web/src/pages/invites/MembersInvitePage.tsx

```diff
@@ -0,0 +1,105 @@
+import { Invite } from '../../design-system/icons';
+import useStyles from '../../design-system/config/text.styles';
+
+export function MembersInvitePage() {
+  const [form] = Form.useForm();
+  const { classes } = useStyles();
+
+  const {
+    data: members,
+    isLoading: loadingMembers,
+    refetch,
+  } = useQuery<any[]>('getOrganizationMembers', getOrganizationMembers);
+  const { isLoading: loadingSendInvite, mutateAsync: sendInvite } = useMutation<
+    string,
+    { error: string; message: string; statusCode: number },
+    string
+  >((email) => inviteMember(email));
+
+  async function onSubmit({ email }) {
+    if (!email) return;
+
+    await sendInvite(email);
+    await refetch();
+    form.resetFields(['email']);
+  }
+
+  return (
+    <PageContainer>
+      <PageHeader
+        title="Team Members"
+        actions={
+          <Form onFinish={onSubmit} form={form}>
+            <Group align="center" spacing={10}>
+              <Form.Item name="email" rules={[{ required: true }]} style={{ marginBottom: 0 }}>
+                <Input required data-test-id="invite-email-field" placeholder="Invite user by email" />
+              </Form.Item>
+              <Button submit icon={<Invite />}>
+                Invite
+              </Button>
+            </Group>
+          </Form>
+        }
+      />
+
+      <Container fluid mt={15} ml={5}>
+        {members?.map((i) => {
+          return (
+            <>
+              <MemberRowWrapper key={i._id}>
+                <Avatar style={{ marginRight: 10, width: 40, height: 40 }} src={i.user?.profilePicture} radius="xl">
+                  {capitalize(i.user?.firstName[0])} {capitalize(i.user?.lastName[0])}
+                </Avatar>
+                <Group direction="column" spacing={5}>
+                  <Text className={classes.heading}>
+                    {i.user
+                      ? `${capitalize(i.user?.firstName as string)} ${capitalize(i.user?.lastName as string)}`
+                      : i.invite.email}
+                  </Text>
+                  <Text className={classes.subHeading}>{i.user?.email ? i?.user.email : null}</Text>
+                </Group>
+                <ActionsSider>
+                  <div style={{ marginLeft: 10 }}>
+                    {/* eslint-disable-next-line no-nested-ternary */}
+                    {i.memberStatus === MemberStatusEnum.INVITED ? (
+                      <Tag>Invite Pending</Tag>
+                    ) : i.roles.find((role: string) => role === 'admin') ? (
+                      <Tag>Admin</Tag>
+                    ) : (
+                      <Tag>Member</Tag>
+                    )}
+                  </div>
+                </ActionsSider>
+              </MemberRowWrapper>
+              <Divider className={classes.seperator} />
+            </>
+          );
+        })}
+      </Container>
+    </PageContainer>
+  );
+}
+
+const AddMemberRow = styled.div`
```

Comment:
Beacause mantine styles are based on emotion, we also prefer that, instead of styled-components. 
Would you mind making that change? There is an example in the [TemplatesListPage](https://github.com/notifirehq/notifire/blob/13f6433647ec829b47332bf4fce2669672f355fc/apps/web/src/pages/templates/TemplatesListPage.tsx#L6)